src/Libraries/Auth/ApiAuth.php

```diff
@@ -164,29 +177,64 @@ public function getUpdatedTokens(array $user)
 
     /**
      * Verify
-     * @param int $code
+     * @param int $otp
+     * @param string $otp_token
      * @return array
      * @throws \Exception
      */
-    public function verify($code)
+    public function verify($otp, $otp_token)
     {
-        $user = (array) $this->user();
+        $user = $this->authService->get($this->keys['otpToken'], $otp_token);
+
+        if (new \DateTime() >= new \DateTime($user[$this->keys['otpExpiryIn']])){
+            throw new AuthException(ExceptionMessages::VERIFICATION_CODE_EXPIRY_IN);
```

Comment:
The constant Quantum\Exceptions\ExceptionMessages::VERIFICATION_CODE_EXPIRY_IN was not found.